app/config/config.yml

```diff
@@ -33,6 +33,13 @@ framework:
     # assets: ~
     php_errors:
         log: true
+    cache:
+        pools:
+            cache.programmes:
+                # cache.adapter.psr6 act as a proxy for the adapter we set on the "provider" section,
+                # this is required when and adapter is not supported directly
+                adapter: cache.adapter.psr6
+                provider: cache.null_provider # locally we don't want to use cache
```

Comment:
We're not going to setup Redis caching for Clifton just yet - it's currently performant using just the DB and we've still got a bunch of questions we need to answer about what the redis box on AWS shall look like.

We'll add the production config to use Redis once we've finalised the config for the cluster and it's up and running.